src/interface/viewport.h

```diff
@@ -41,7 +41,7 @@ enum {
 	VIEWPORT_FLAG_HIDE_BASE = (1 << 12),
 	VIEWPORT_FLAG_HIDE_VERTICAL = (1 << 13),
 	VIEWPORT_FLAG_INVISIBLE_SPRITES = (1 << 14),
-	VIEWPORT_FLAG_15 = (1 << 15)
+	VIEWPORT_FLAG_HIDE_TOP_BOTTOM_TOOLBAR = (1 << 15),
```

Comment:
I don't think the state of the toolbar should be saved as a viewport flag anyway. Its not related to viewports. A global variable would be better.